src/models/dailySurveys.js

```diff
@@ -0,0 +1,98 @@
+const Sequelize = require('sequelize');
+const { sequelizeConnection, DataTypes } = require('./index.js');
+
+const allSurveys = sequelizeConnection.define('dailySurveys', {
```

Comment:
Oh ok, so this is for aggregate report or the day? We wouldn't create a table for that in the database, we would create a report by running a query on the database.